docs/python/readme/requirements.rst

```diff
@@ -0,0 +1,11 @@
+.. _requirements:
+
+Requirements
+============
+
+* Python 3.11+
+* `buf <https://buf.build/docs/introduction>`_
+* requirements.txt
+
+.. note::
```

Comment:
Quick question, If I modify the README to have this in the quick_start section, I will also have to edit this file correct?

Reply:
Yes, that's correct, you need to edit the corresponding .rst file, I believe there is no way to pull the content from the README automatically but I'll do some research and if find something I'll let you know